README.md

```diff
@@ -43,14 +48,24 @@ bash run_pipeline.sh
 
 For exact instructions on how to reproduce our analysis see `run_pipeline.sh`.
 
+## Results
+
+Our _in silico_ deconvolution of CD4+ cells, CD8+ cells, and Macrophages in TCGA
+data matches very closely to immunohistochemistry estimates of the same cell types
+in a separate dataset. The proportions of immune cell infiltrate across subtypes
+corresponds strongly.
+
+![immune deconvolution and IHC](figures/boxplot_validation_TCGA_summary.png?raw=true)
+
+We also observed that high macrophage infiltration was associated with worse outcomes
+in the TCGA dataset. This relationship was strengthed after adjusting for several
+covariates including age, gender, and gene expression based subtype.
+
+![TCGA macrophage survival](figures/TCGA_kaplanmeier_Macrophages.png?raw=true)  
+
+
 ## Contact
 
 For all code related questions, bug reporting, or feature requests please file a
 [GitHub issue](https://github.com/greenelab/GBM_immune_profiles/issues "file an issue")
 
-## Dependencies
-
-All code was performed in R version 3.2.3 and packages were managed by the
-[checkpoint](https://cran.r-project.org/web/packages/checkpoint/index.html "Microsoft checkpoint r package")
```

Comment:
> there shouldn't be a need for someone else to rebuild the docker image

disagree... at some point it's likely that you or others need to rebuild, because they need to add a new dependency as the project evolves.

Also if you don't specify versioning in the Dockerfile, where do you go to see which versions of everything got installed?